### PR TITLE
Check PHQ-9

### PR DESCRIPTION
**Pull request by ibevers**
Source link:
https://www.psychiatry.org/getmedia/a3986be5-94af-42e7-afce-19234c2f4998/APA-DSM5TR-SeverityMeasureForDepressionAdult.pdf

### CODE REVIEW

**Comment by ibevers** [comment]
Screenshot 2024-05-09 at 6 37 38 PM
New validation passes

**Comment by satra** [comment]
@ibevers - item 10 should not be removed. that's in the PHQ9. the APA reference is not the primary reference for PHQ9. it may be useful to track down the original reference/paper for these instruments.

**Comment by satra** [comment]
the APA version says: "Adapted from Patient Health Questionnaire—9 (PHQ-9) for research and evaluation purposes". we do want the questionnaires to be as they were published. one could create an APA version of PHQ9, but we should not need to.

**Comment by ibevers** [comment]
@satra Okay, I will track down the original and update accordingly.

**Comment by ibevers** [comment]
@satra here is the original paper, as far as I can tell:
[PHQ9_paper_spitzer1999.pdf](https://github.com/ReproNim/reproschema-library/files/15325429/PHQ9_paper_spitzer1999.pdf)

**Comment by ibevers** [comment]
@satra
Screenshot 2024-05-15 at 2 37 19 PM

**Comment by satra** [comment]
when you have a link can you add it to the citation for the activity schema? i just filed an issue to update the citation key.

File: activities/PHQ-9/items/phq9_10 [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
-    "@context": "https://raw.githubusercontent.com/ReproNim/reproschema/1.0.0-rc4/contexts/generic",
+    "@context": "https://raw.githubusercontent.com/ReproNim/reproschema/1.0.0-rc1/contexts/generic",
```
Comment (satra) [inline activities/PHQ-9/items/phq9_10:2]: this switched back to rc1, here and elsewhere.

Comment (ibevers) [reply activities/PHQ-9/items/phq9_10]: Resolved

**Comment by ibevers** [comment]
> when you have a link can you add it to the citation for the activity schema? I just filed an issue to update the citation key.
Currently, the citation value must be a string. Are you saying that resolution of the issue will allow lists of citations and that I should append the citation I found to the list when that is allowed?

I didn't see your issue on: https://github.com/ReproNim/reproschema
Where is it?

**Comment by satra** [comment]
https://github.com/ReproNim/reproschema/pull/456#issuecomment-2113233631

**Comment by ibevers** [comment]
> @satra here is the original paper, as far as I can tell: [PHQ9_paper_spitzer1999.pdf](https://github.com/ReproNim/reproschema-library/files/15325429/PHQ9_paper_spitzer1999.pdf)

This paper included questions about anxiety as well as the PHQ-9 questions, so I am not updating the citation, even though it is earlier than the 2001 paper where the PHQ-9 was used separately.

**Comment by satra** [comment]
use this: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1495268/

**Comment by ibevers** [comment]
> use this: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1495268/

That is the one being used.

**Pull request merged**